tests/xml2/XMLBearTest.py

```diff
@@ -79,6 +79,20 @@ def load_testdata(filename):
     settings={'xml_dtd': dtd_url},
     tempfile_kwargs={'suffix': '.xml'})
 
+XMLBearStyleTest = verify_local_bear(
+    XMLBear,
+    valid_files=(valid_xml_file,),
+    invalid_files=(invalid_xml_chars,),
+    settings={'xml_style': 'oldxml10'},
+    tempfile_kwargs={'suffix': '.xml'})
+
+XMLBearInvalidStyleTest = verify_local_bear(
+    XMLBear,
+    valid_files=(valid_xml_file,),
+    invalid_files=(invalid_xml_chars,),
+    settings={'xml_style': 'wrong args'},
```

Comment:
need to confirm that this error is the cause of the failure.
otherwise, some other error could occur and this test look like it has been effective.